plugins/telescope/default.nix

```
@@ -20,7 +20,21 @@ helpers.neovim-plugin.mkNeovimPlugin config {
   deprecateExtraOptions = true;
   optionsRenamedToSettings = [ "defaults" ];
 
-  imports = [ ./extensions ];
+  imports = [
+    ./extensions
+
+    # TODO introduced 2024-05-24: remove 2024-08-24
+    (mkRemovedOptionModule
+      [
+        "plugins"
+        "telescope"
+        "keymapsSilent"
+      ]
+      ''
+        This option no longer has any effect now that the `plugin.telescope.keymaps` implementation uses `<cmd>`.
+      ''
```

```suggestion
      "This option no longer has any effect now that the `plugin.telescope.keymaps` implementation uses `<cmd>`."
```

Just for the sake of requesting a change X)

---

(I would understand if you told me that you didn't care haha)

---

No problem, done